content/blogposts/2024/coding-on-the-road-with-android-cody.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Coding on the road with Android and Cody"
+authors:
+  - name: Tino Wening
+    url: https://twitter.com/PriNova75
+  
+publishDate: 2024-02-06T10:00-07:00
+description: "This post is a fun and informal guide for digital nomads, showcasing how Termux and Cody turn your tablet into a beach-friendly coding hub. It's all about ditching the laptop for a lightweight setup with Termux's Linux shell, VS Code's web version, and Cody."
+tags: [blog]
+slug: "coding-on-the-road-with-android-cody"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-on-the-road/coding-on-the-road-with-android-cody-og.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-on-the-road/coding-on-the-road-with-android-cody-og.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+**Hey there, fellow Digital Nomads!** Craving that perfect blend of beachside cocktails and coding deadlines? Ever wished your hotel room came with a built-in IDE, not just a mini bar? Well, strap on your walking shoes and ditch the laptop, because we're about to unlock the secret to mobile coding freedom: Termux and Cody.
```

Comment:
Can we link to the Termux homepage as well as /cody when we mention these two at the end.

Reply:
done!